source/_posts/2023-03-18-hexo-7-0-0-rc1-released.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Hexo 7.0.0 (RC1) Released
+---
+
+> [GitHub Release Note: v7.0.0 (RC1)](https://github.com/hexojs/hexo/releases/tag/v7.0.0-rc1)
+
+## Migration Guid
+
+Some of tag feature has been droped. Please use [hexo-tag-embed](https://github.com/hexojs/hexo-tag-embed) instead of them.
+
+```sh
+$ npm i hexo-tag-embed
+```
+
+> **Note**
+>
+> No need to install if you are not using (or will not use) `gist`, `youtube`, `jsfiddle`, `vimeo` tags in your post or page.
+
+## Breaking Changes
+
+* chore: require node14+ by [@yoshinorin] in [#5061](https://github.com/hexojs/hexo/pull/5061)
+* Dropped tag features. Please see `Migration Guid` section.
+    * refactor: drop `gist` tag by [@yoshinorin] in [#5067](https://github.com/hexojs/hexo/pull/5067)
+    * refactor: drop `youtube` tag by [@yoshinorin] in [#5064](https://github.com/hexojs/hexo/pull/5064)
+    * refactor: drop `jsfiddle` tag by [@yoshinorin] in [#5066](https://github.com/hexojs/hexo/pull/5066)
+    * refactor: drop `vimeo` tag by [@yoshinorin] in [#5065](https://github.com/hexojs/hexo/pull/5065)
+* Dropped features
+    * refactor: drop `external_link` boolean type by [@yoshinorin] in [#5063](https://github.com/hexojs/hexo/pull/5063)
+    * refactor: drop `use_date_for_updated` option for `updated_option` by [@yoshinorin] in [#5062](https://github.com/hexojs/hexo/pull/5062)
```

Comment:
We should also mention how to migrate these in the migration guide.

Reply:
?

Just install [hexo-tag-embed](https://github.com/hexojs/hexo-tag-embed). So, I think we do not need to add more explanation in the migration guide. What do you think?